manim/mobject/graphing/coordinate_systems.py

```diff
@@ -402,10 +383,22 @@ def construct(self):
         :class:`~.VGroup`
             A :class:`~.Vgroup` of the labels for the x_axis and y_axis.
 
-        See Also
+        .. seealso::
+            :class:`get_x_axis_label`
+            :class:`get_y_axis_label`
+
```

Comment:
[This](https://manimce--2570.org.readthedocs.build/en/2570/reference/manim.mobject.graphing.coordinate_systems.CoordinateSystem.html#manim.mobject.graphing.coordinate_systems.CoordinateSystem.get_axis_labels) doesn't render correctly

Reply:
Admonitions must have two newline characters after any section. Changes done in [e4d6dd3](https://github.com/ManimCommunity/manim/pull/2570/commits/e4d6dd35b15007c1d1643cf2908b28d235f2f16d)